src/components/molecules/UserStatusManager/components/UserStatusPanel/UserStatusPanel.scss

```diff
@@ -0,0 +1,33 @@
+@import "scss/constants";
+
+$picker-icon--size: 35px;
+$delete-icon--size: 25px;
+
+.UserStatusPanel {
+  &__status {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 5px 0;
```

Comment:
looks like `5px` spacing is repeating here (can we create a variable for that?) (1 / 2)

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/6db8a73718475d1f8e780610acf3becd6f5e4c3b